indigo_api/templates/indigo_api/akn/coverpage_act.html

```diff
@@ -55,27 +55,124 @@ <h2>{{ subt }}</h2>
         <li class="commencement-date">
           {% blocktrans %}Not commenced{% endblocktrans %}
         </li>
-      {% elif not document.commencement_date %}
+      {% elif document.work.main_commencement and document.work.main_commencement.all_provisions %}
+        {% with document.work.main_commencement as commencement %}
+        {% with commencement.commencing_work as commencing_work %}
+        {% if not commencement.date %}
+          <li class="commencement-date">
+            {% blocktrans %}Commencement date unknown{% endblocktrans %}
+              {% if commencing_work %}
```

Comment:
Over-indented.

Reply:
Would it be easier to merge the has-date and no-date logical blocks together, and then do the different behaviour right at the last minute? It would avoid duplicating all the numbered title and resolver stuff.